[Feature] New task plugin Pytorch (#11498)
apache/dolphinscheduler: docs/docs/en/guide/task/pytorch.md

# Pytorch Node (experimental)

## Overview

[Pytorch](https://pytorch.org) is a mainstream Python machine learning library.

> In order to enable users to **run Pytorch project in DolphinScheduler more convenient,** Pytorch task plugin is implemented. It provides convenient Python environment management and supports the running of Python projects.

Reviewer: ```suggestion
`Pytorch` task plugin enables users to run `Pytorch` projects in DolphinScheduler more conveniently. In addition, it supports handy Python environment management.
```

Reviewer: Please avoid using passive voice unless necessary in tech doc writing.

Author: I see. Thank you. I'll be careful in the future.

> Unlike the Python task plugin, this component allows users to **quickly use existing Python environments or create new ones (using Virtualenv or Conda)**. **Support for running Python projects (native or Git projects)** instead of just Python scripts.

Reviewer: ```suggestion
`Pytorch task plugin` is more than `Python task plugin`, which allows users to **conveniently use existing Python environments or create new ones (using `Virtualenv` or `Conda`)**. **It supports running Python projects (native or Git projects)** instead of just Python scripts.
```
